### PR TITLE
BaSP-TG-92: Set Up Firebase

### PR DESCRIPTION
**Pull request by PilarFernandezC**
- Install Firebase
- Create "helpers/firebase/index.js" inside src folder
- Update .ENV with Firebase credentials
- Create PrivateRoutes.js file in routes folder
- Create Auth folder with Login/index.js and SignUp/index.js inside of it
- In redux folder create auth folder with actions.js, constants.js, reducer.js and thunks.js inside of it
- Create actions.js, constants.js, reducer.js and thunks.js

**Contributors:** @mara666 @Maxi-SR @DamianPalavecino

### CODE REVIEW

File: src/redux/auth/thunks.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { loginRejected, loginPending, logoutRejected, logoutPending } from './actions';
+
+import { signInWithEmailAndPassword, signOut } from 'firebase/auth';
+import { auth } from '../../Helpers/firebase';
```
Comment (Maxi-SR) [inline src/redux/auth/thunks.js:4]: It's possible to import with` 'Components/Helpers/firebase'`  ?

**Review by mara666 — APPROVED** [review]
Good job!

File: src/redux/auth/thunks.js (hunk 1 above)
Comment (PilarFernandezC) [reply src/redux/auth/thunks.js]: Done

**Review by Maxi-SR — APPROVED** [review]

File: src/redux/auth/thunks.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ import { auth } from 'Components/Helpers/firebase';
 export const login = (inputData) => {
   return async (dispatch) => {
     dispatch(loginPending());
+    console.log('esto es una prueba');
```
Comment (Juanoauil) [inline src/redux/auth/thunks.js:9]: please remove console.log

Comment (PilarFernandezC) [reply src/redux/auth/thunks.js]: Done

**Pull request merged**